Makefile

```
@@ -0,0 +1,37 @@
+
+test:
+	@go test ./...
+
+clean: # Cleanup build artifacts
+	@rm -rf dist/*
+
+build: clean ## Build the plugin package
+	@mkdir -p dist/
+	@go build -o dist/plugin main.go
+
+run: build ## Execute the Concom agent with the built plugin
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)